metaflow/plugins/kubernetes/kubernetes_cli.py

```diff
@@ -84,6 +84,12 @@ def kubernetes():
     default=5 * 24 * 60 * 60,  # Default is set to 5 days
     help="Run time limit in seconds for Kubernetes pod.",
 )
+@click.option(
+    "--tolerations",
+    default=None,
+    type=JSONTypeClass(),
+    multiple=False,
```

Comment:
Shouldn't this be `true`? 

`secrets` and `node_selector` also have `multiple=true`.

Reply:
I decided to use `JSONType` because it is easy to pass an object.
Or do you mean to use something like `--tolerations='{"key": "arch", "operator": "Equal", "value": "amd"}' --tolerations='{"key": "foo", "operator": "Equal", "value": "bar"}'`?

Reply:
So, I think if there are multiple tolerations, they'd look something like:
`[{"key": "arch", "operator": "Equal", "value": "amd"}, {"key": "foo", "operator": "Equal", "value": "bar"}]`

If that's correct, then your approach is totally valid.

Reply:
yes, correct. The input is an array of objects